CITATION.cff

```diff
@@ -1,40 +1,75 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
-url: "https://github.com/helmholtz-analytics/heat"
+title: HeAT
+message: 'If you use this software, please cite it as below.'
+type: software
+authors:
+  - family-names: Götz
+    given-names: Markus
+  - family-names: Debus
+    given-names: Charlotte
+  - family-names: Coquelin
+    given-names: Daniel
+  - family-names: Krajsek
+    given-names: Kai
+  - family-names: Comito
+    given-names: Claudia
+  - family-names: Knechtges
+    given-names: Philipp
+  - family-names: Hagemeier
+    given-names: Björn
+  - family-names: Tarnawa
+    given-names: Michael
+  - family-names: Hanselmann
+    given-names: Simon
+  - family-names: Siggel
+    given-names: Martin
+  - family-names: Basermann
+    given-names: Achim
+  - family-names: Streit
+    given-names: Achim
```

Comment:
This block causes zenodo to always use this list of authors for each release (we want to have a list of actual contributors instead), and also somehow messed up the versioning on zenodo, see below. That's why at the time we decided to remove it.

![Screenshot 2023-12-19 at 09 52 39](https://github.com/helmholtz-analytics/heat/assets/39374113/ff5dcfdf-5ac0-40dc-b9ed-532fd4c908cb)

Reply:
OK, here's the link to the previous cff update. I see the versioning problem was actually do to us hardcoding the version and release date in the very first version of the citation file. So that should no longer arise. Still, I would prefer to have the actual contributors as authors of a release, instead of the authors of the 2020 paper.

https://github.com/helmholtz-analytics/heat/pull/1088